deltachat-ios/Helper/ShortcutManager.swift

```diff
@@ -80,7 +80,7 @@ public class ShortcutManager {
         }
 
         let infoDict = message.getWebxdcInfoDict()
-        let iconData = scaledDownLogo?.pngData() ?? UIImage(named: "appicon")?.pngData() ?? nil
+        let iconData = scaledDownLogo?.pngData() ?? nil
```

Comment:
a fallback beside "nil" for "no webxdc icon" is not needed - core already provides a fallback (the webxdc icon)